src/pkg/packaging/rpm/dotnet-runtime-deps-rpm_config_rhel.7-x64.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Who is going to use our RHEL7 rpm vs. the official RHEL package that RedHat produces?  Do we really need our own RHEL7 rpm?

/cc @leecow

Reply:
For final releases I would expect RHEL customers to obtain updates through the SCL machinery. However, if they also want to experiment with daily builds or Previews, our packages are currently the only solution.